packages/shared/lib/clients/orchestrator.ts

```diff
@@ -110,9 +109,13 @@ export class Orchestrator {
             ...(activeSpan ? { childOf: activeSpan } : {})
         });
         const startTime = Date.now();
-        const workflowId = `${SYNC_TASK_QUEUE}.ACTION:${actionName}.${connection.connection_id}.${uuid()}`;
         try {
-            await logCtx.info(`Starting action workflow ${workflowId} in the task queue: ${SYNC_TASK_QUEUE}`, { input });
+            await logCtx.info(`Starting action '${actionName}'`, {
+                input,
+                action: actionName,
+                connection: connection.connection_id,
+                integration: connection.provider_config_key
```

Comment:
those 2 syncs are already logged in the operation (applicable for multiple lines)

Reply:
can we easily show them in the message payload then?

Reply:
they are shown on top of the Operation drawer

Reply:
yes but this is to also show this info in the individual message payload.

Reply:
ideally we don't store those information again just for display reason, you can pass the operation to the message drawer if needed

Reply:
got it. Let me remove them. This info is not absolutely required in the message drawing for now